VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-toastr.service.ts

```diff
@@ -108,7 +108,12 @@ export class NotificationToastrService {
     reportPoorConnection(heartbeat: ParticipantHeartbeat) {
         const heartbeatKey = `${heartbeat.participantId}_${heartbeat.heartbeatHealth.toString()}`;
         if (this.activeHeartbeatReport.indexOf(heartbeatKey) >= 0) {
-            return;
+            this.activeHeartbeatReport.push(heartbeatKey);
+            if (this.activeHeartbeatReport.filter(x => x.indexOf(heartbeatKey) >= 0).length > 25) {
```

Comment:
can we setup a timer on 2 min interval to clear the array?

Reply:
done